feature/platform/tests/power_admin_down_up_test/power_admin_down_up_test.go

```diff
@@ -24,18 +24,34 @@ func TestFabricPowerAdmin(t *testing.T) {
 	dut := ondatra.DUT(t, "dut")
 	fs := components.FindComponentsByType(t, dut, oc.PlatformTypes_OPENCONFIG_HARDWARE_COMPONENT_FABRIC)
 
+	removable_fabrics := make([]string, 0)
 	for _, f := range fs {
-		t.Run(f, func(t *testing.T) {
+		compMtyVal, compMtyPresent := gnmi.Lookup(t, dut, gnmi.OC().Component(f).Empty().State()).Val()
```

Comment:
You still have two versions of each check in this function (twice check on empty, and twice check on removable).

This doesn't make sense to me. Each check should be present only once

Reply:
Resolved